code/game/mecha/mecha.dm

```diff
@@ -145,7 +146,7 @@
 /obj/mecha/get_cell()
 	return cell
 
-/obj/mecha/Destroy()
+/obj/mecha/deconstruct(disassembled = TRUE)

```

Comment:
deconstruct calls qdel which calls destroy
Without this change, exporting mechs makes them explode into ruined mechs once they arrive at central.